Ginger/Ginger/SourceControl/CheckInPage.xaml.cs

```diff
@@ -240,8 +240,11 @@ private async void CheckInButton_Click(object sender, RoutedEventArgs e)
                                 }
                                 else if (!fi.Locked)
                                 {
+                                    RepositoryFolderBase repositoryFolderBase = WorkSpace.Instance.SolutionRepository.GetRepositoryFolderByPath(Path.GetDirectoryName(fi.Path));
```

Comment:
Need to make it generic as this needs to be called also in other places where update file is needed.
Also test with Git